books_to_scrape/items.py

```diff
@@ -0,0 +1,18 @@
+# Define here the models for your scraped items
+#
+# See documentation in:
+# https://docs.scrapy.org/en/latest/topics/items.html
+
+import scrapy
+from dataclasses import dataclass
+
+
+@dataclass
+class BooksToScrapeItem(scrapy.Item):
+    title: str
+    price: float
+    amount_in_stock: int
+    rating: int
+    category: str
+    description: str
+    upc: str
```

Comment:
Each attribute should be defined using `scrapy.Field()`. For example, `title = scrapy.Field()`. This is necessary for Scrapy to properly handle the item fields.